### PR TITLE
add wikipedia biography dataset

### PR DESCRIPTION
**Pull request by ACR0S**
My first PR containing the Wikipedia biographies dataset. I have followed all the steps in the [guide](https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md). It passes all the tests.

### CODE REVIEW

**Comment by ACR0S** [comment]
Does anyone know why am I getting this "Some checks were not successful" message? For the _code_quality_ one, I have successfully run the flake8 command.

**Comment by ACR0S** [comment]
Ok, I need to update the README.md, but don't know if that will fix the errors

**Comment by yjernite** [comment]
Hi @ACR0S , thanks for adding the dataset!

It looks like `black` is throwing the code quality error: you need to run `make style` with the latest version of `black` (`black --version` should return 20.8b1)

We also added a requirement to specify encodings when using the python `open` function (line 163 in the current version of your script)

Finally, you will need to add the tags and field descriptions to the README as described here https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Let us know if you have any further questions!

**Comment by yjernite** [comment]
Also, please leave the full template of the readme  with the `[More Information Needed]` paragraphs: you don't have to fill them out now but it will make it easier for us to go back to later :)

**Comment by ACR0S** [comment]
Thank you for your help, @yjernite! I have updated everything (finally run the _make style_, added the tags, the ecoding to the _open_ function and put back the empty fields in the README). Hope it works now! :)

**Comment by yjernite** [comment]
LGTM!

File: datasets/wiki_bio/README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+---
+YAML tags:
```
Comment (lhoestq) [inline datasets/wiki_bio/README.md:2]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Awesome thank you for adding this one !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**